src/libs/mergeRefs.ts

```diff
@@ -0,0 +1,19 @@
+import {LegacyRef, MutableRefObject, RefCallback} from 'react';
+
+/**
+ * Assigns element reference to multiple refs.
+ * Source: https://github.com/gregberge/react-merge-refs/blob/main/src/index.tsx
```

Comment:
I think this link should be removed.

I don't think it brings much value, while it might raise licensing questions.

Reply:
Do you know if we have any guidelines/processes on what to do in such case? 

What licensing questions do you have in mind? The code is taken from there, with the same notion as is [compose](https://github.com/Expensify/App/blob/6a55d16af3e67d4e708287da84bb99c46f628846/src/libs/compose.ts#L20) taken from Redux.

Reply:
Linking another project this way could be interpreted as directly stating that we create a derivative work, which means we should follow its license. In this case it's MIT, which requires us to copy-paste some copyright clause.

Such simple functions are arguably non-copyrightable, so it doesn't mean we really have to follow the license...

Some people think that the easiest way to avoid this legal topic is not to link sources for trivial utilities "borrowed" from other places on the Internet.

Another argument for it is that it shifts the responsibility towards the PR author.

Reply:
Understood, thanks for the explanation!